src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/PanelStatus.tsx

```diff
@@ -6,21 +6,25 @@ import { getExecutionPoint } from "../../../selectors";
 
 export function PanelStatus({
   analysisPoints,
+  isHot,
   prefixBadge,
 }: {
   analysisPoints: AnalysisPayload;
+  isHot: boolean;
   prefixBadge: string;
 }) {
   const executionPoint = useSelector(getExecutionPoint);
   const isIndexed = useSelector(getIsIndexed);
   const theme = useSelector(getTheme);
   let status = "";
-  let maxStatusLength = 0;
+  let maxStatusLength = "Loading".length;
```

Comment:
"Indexing" is longer than "Loading" 😁  Seems like maybe we should just wait and measure the status length after we've set the status string? (Why are we pre-measuring the max? Now that we're using CSS `width` rather than `minWidth` anyway– maybe this can just be changed too?)

Reply:
It would be nice to replace Indexing with Loading in the UI since we're transitioning to a simpler timeline as well

Reply:
> Why are we pre-measuring the max?

My goal with pre-measuring the max was to avoid the length of the pill/timeline from jumping. I agree that it's awkward, but a better solution did not seem immediately obvious to me. Good call on the loading vs. indexed thing though. Re: changing the actual word in the UI... that's an interesting point! I wonder why we have distinguished between those in the past. I can't think of a compelling reason off of the top of my head right now.